Mathlib/SetTheory/Cardinal/Continuum.lean

```diff
@@ -166,15 +166,22 @@ theorem continuum_mul_ofNat {n : ℕ} [Nat.AtLeastTwo n] : 𝔠 * no_index (OfNa
 
 
 @[simp]
-theorem aleph0_power_aleph0 : aleph0.{u} ^ aleph0.{u} = 𝔠 :=
+theorem aleph0_power_aleph0 : ℵ₀ ^ ℵ₀ = 𝔠 :=
```

Comment:
My understanding was that the notation isn't used here because it isn't universe polymorphic. Is that not true?

Reply:
No, the notation is in fact universe polymorphic.
```
import Mathlib
open Cardinal
#check (ℵ₀ : Cardinal.{5})
```

Reply:
Cool thanks!